### PR TITLE
fix: solved the previous pr's issues

### PR DESCRIPTION
**Pull request by RavenDuffy**

### CODE REVIEW

File: cms/src/admin/app.example.tsx [hunk 1]
```diff
@@ -1,37 +1,8 @@
-import type { StrapiApp } from '@strapi/strapi/admin';
+import type { StrapiApp } from "@strapi/strapi/admin"
 
 export default {
   config: {
-    locales: [
```
Comment (SAndersonCheung) [inline cms/src/admin/app.example.tsx:5]: where have the locales gone

File: frontend/helpers/strapi.ts [hunk 2]
```diff
@@ -47,8 +47,8 @@ export const getAllByType = async (contentType: string, params = {}) => {
       },
     })
 
-    if (query && (query?.data !== null || query?.data?.length > 0)) {
-      for (const entry of query.data) entries.push(entry)
+    if (query && query?.data?.length > 0) {
+      query.data.forEach((entry) => entries.push(entry))
```
Comment (SAndersonCheung) [inline frontend/helpers/strapi.ts:51]: you said this would be a map

File: cms/src/admin/app.example.tsx (hunk 1 above)
Comment (RavenDuffy) [reply cms/src/admin/app.example.tsx:5]: they weren't there to begin with

**Review by SAndersonCheung — APPROVED** [review]
😬

**Pull request merged**